Tiltfile

```diff
@@ -101,12 +101,12 @@ docker_build_with_restart(
     dockerfile_contents="""FROM golang:1.20.8
 RUN apt-get -q update && apt-get install -qyy curl jq
 RUN go install github.com/go-delve/delve/cmd/dlv@latest
-COPY bin/pocketd /usr/local/bin/pocketd
+COPY bin/poktrolld /usr/local/bin/pocketd
```

Comment:
@Olshansk this addresses the issue with hot-reloading actually updating the sequencer on LocalNet. B/c currently, Tilt watches `pocketd` binary that never updates.

Reply:
Let's update everything else.

![Screenshot 2023-11-03 at 1 11 53 PM](https://github.com/pokt-network/poktroll/assets/1892194/72049ced-1025-4794-8ea8-6dd5881140cb)

Reply:
@Olshansk I did not update everything - only mentions of the binary itself. There are still other places where pocketd is mentioned, and we can change that, but I want to do this in a aseparate PR along with changes to the helm chart.